src/reporter/main.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+# * coding: utf8 *
+"""
+a description of what this module does.
+this file is for testing linting...
+"""
+
+import logging
+import sys
+from pathlib import Path
+
+from . import reports
+
+
+def run_reports(logger):
+    """
+    Main logic for instantiating report objects and running their methods.
+    """
+
+    agol_out_path = Path(r'c:\temp\report_out_test.csv')
```

Comment:
**nitpick**: move this hardcoded value to configuration or an input. Maybe write this to a drive sheet instead so everyone can view it?

Reply:
I like the scheduling and publishing results idea. I've been toying with the idea of saving it into a database where each item gets its own table and each row is one report's results. We schedule it weekly or monthly, and then we can watch usage change over time. 

Or we could overwrite the data in an AGOL hosted table on every run so that we can load it directly into an Esri dashboard (cluttering AGOL with hundreds of tables like the db above wouldn't be ideal).

Reply:
I've added an issue for publishing the results to a database/google sheet for future consideration.